gcsfs/core.py

```diff
@@ -301,6 +315,7 @@ def __init__(
         endpoint_url=None,
         default_location=None,
         version_aware=False,
+        experimental_zb_hns_support=False,
```

Comment:
From the definition of `__new__`, this must always be False? This is here to stop the kwarg leaking maybe.